### ISSUE TITLE
Error with downloading SHASUM

### ISSUE THREAD

**Issue by alexellis** [inlets/inletsctl#84]
Error with downloading SHASUM

## Expected Behaviour

Download script would work without error and check the SHASUM of the tgz

## Current Behaviour

A user reported that it wasn't working.

```bash
alex@kosmos github.com % curl -SLs https://inletsctl.inlets.dev|sudo bash
Downloading package https://github.com/inlets/inletsctl/releases/download/0.7.0/inletsctl-darwin.tgz as /tmp/inletsctl-darwin.tgz
Download Complete, extracting /tmp/inletsctl-darwin.tgz to /tmp/ ...
OK
SHA256 fetched from release: Not
shasum: standard input: no properly formatted SHA checksum lines found
SHA mismatch! This means there must be a problem with the download
```

It seems that we are now missing the SHAs on the releases page from 0.7.0 - https://github.com/inlets/inletsctl/releases/tag/0.7.0

If you need to download inletsctl, just head over to the releases page for the time being.

**Comment by alexellis** [comment]
@isabellaliu77 FYI we ran into this issue after merging #82

**Comment by alexellis** [comment]
Fixed in 8641a9a09903f0e3b18a4ed1d388a35663a52dc4

```
bash-3.2$  curl -SLs https://inletsctl.inlets.dev |sudo bash
Password:
Downloading package https://github.com/inlets/inletsctl/releases/download/0.7.1/inletsctl-darwin.tgz as /tmp/inletsctl-darwin.tgz
Download Complete, extracting /tmp/inletsctl-darwin.tgz to /tmp/ ...
OK
SHA256 fetched from release: 426706c84e9c9b46869fc08a5dfad28f299559f977c50055d074553467f2f1ee

Running with sufficient permissions to attempt to move inletsctl to /usr/local/bin
New version of inletsctl installed to /usr/local/bin
 _       _      _            _   _ 
(_)_ __ | | ___| |_ ___  ___| |_| |
| | '_ \| |/ _ \ __/ __|/ __| __| |
| | | | | |  __/ |_\__ \ (__| |_| |
|_|_| |_|_|\___|\__|___/\___|\__|_|

Version: 0.7.1
Git Commit: 8641a9a09903f0e3b18a4ed1d388a35663a52dc4
```
